### ISSUE TITLE
Diffifculty in installing 

### ISSUE THREAD

**Issue by yogiii363** [vitorfs/bootcamp#163]
I am having some difficulties in the django-channels and redis step. Can someone explain me stepwise what I need to to for that?

**Comment by sebastian-code** [comment]
Its hard to follow your need @yogiii363 because is not clear where exactly your are having an issue. If you follow the instructions in the Wiki page you'll be able to use Bootcamp. But you're right on something. Those steps require some basic knowledge on Redis and django-channels, but if you use the docker-compose template, you'll be able to deploy without a hassle.

Please elaborate further your case to see how we can help you.

**Comment by sebastian-code** [comment]
Closing.
